radix-engine-common/src/constants/wasm.rs

```diff
@@ -13,5 +13,11 @@ pub const MAX_NUMBER_OF_GLOBALS: u32 = 512;
 /// The max number of functions
 pub const MAX_NUMBER_OF_FUNCTIONS: u32 = 64 * 1024;
 
+/// The max number of function parameters
+pub const MAX_NUMBER_OF_FUNCTION_PARAMS: u32 = 25;
```

Comment:
Seems like an arbitrary number, maybe set it to be some base2 instead like `32`?

Reply:
Sure